app/services/foreman_templates/template_importer.rb

```diff
@@ -25,6 +25,21 @@ def initialize(args = {})
     end
 
     def import!
+      if @repo.start_with?('http://', 'https://', 'git://')
+        import_from_git
+      elsif repo.start_with?('/')
```

Comment:
Yeah, `~` is useful, good point. Falling back to file-based sounds reasonable - the error message can become `Using file-based import, but couldn't find #{File.expand_path(@repo)}` (or something like that) to show where the code is searching. 

+1 for a manual, but you might want at least a short note in the README for now, until that gets written. Don't spend too long on it ;)